cmd/kops/delete_instancegroup.go

```diff
@@ -61,9 +61,9 @@ func NewCmdDeleteInstanceGroup(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "instancegroup",
 		Aliases: []string{"instancegroups", "ig"},
-		Short:   delete_ig_short,
-		Long:    delete_ig_long,
-		Example: delete_ig_example,
+		Short:   deleteIgShort,
```

Comment:
FYI, Go prefers `deleteIGShort`.  https://github.com/golang/go/wiki/CodeReviewComments#initialisms

I don't like it personally, but that's the GOld standard